src/accordion/Accordion.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @overview An Accordion module comprised of sections that toggle open and close
+ * @module Accordion.js
+*/
+
+export const Accordion = {
+
+  /**
+   * Inits accordion elements
+   * @returns {void}
+  */
+  init() {
+    this.accordionSection  = document.getElementsByClassName(`js-accordion-section`);
```

Comment:
Do you think we should just use .querySelector/.querySelectorAll to standardize to be more like the $() in jQuery? .getElementById seems right for ID targeting, otherwise I think querySelector is more modern and practical.